date-input.html

```diff
@@ -22,12 +22,14 @@
 
     input[is="iron-input"], span {
       text-align: center;
+      min-width: 0;
```

Comment:
Why don't we fix this in `<paper-input-container>` where the rest of the input styling is normalized?